src/plugins/vis_type_vega/public/vega_view/vega_map_view/view.ts

```diff
@@ -68,11 +68,18 @@ export class VegaMapView extends VegaBaseView {
 
   private getMapParams(defaults: { maxZoom: number; minZoom: number }): Partial<MapboxOptions> {
     const { longitude, latitude, scrollWheelZoom } = this._parser.mapConfig;
-    const zoomSettings = validateZoomSettings(this._parser.mapConfig, defaults, this.onWarn);
+    const { zoom, maxZoom, minZoom } = validateZoomSettings(
```

Comment:
This change partially fixes #17212.  Setting the `restoreSignalValuesOnRefresh` parameter to` true` restores the previous values of `zoom`,` longitude` and `latitude`.

https://user-images.githubusercontent.com/20072247/107494554-fc316600-6b9f-11eb-8817-c5089a53346c.mov

Reply:
@alexwizp can you share the spec you were testing with? I'd like to see how it works and maybe improve that section of the docs.

Reply:
you can use `[Flights] Airport Connections (Hover Over Airport)`. Just set `restoreSignalValuesOnRefresh` to `true`